frontend/talentsearch/src/js/components/search/SearchContainer.test.tsx

```diff
@@ -0,0 +1,59 @@
+/**
+ * @jest-environment jsdom
+ */
+import "@testing-library/jest-dom";
+import { screen } from "@testing-library/react";
+import React from "react";
+import { fakeClassifications, fakeCmoAssets } from "@common/fakeData";
+
+import { render } from "../../tests/testUtils";
+import { SearchContainer } from "./SearchContainer";
+import type { SearchContainerProps } from "./SearchContainer";
+
+const mockClassifications = fakeClassifications();
+const mockCmoAssets = fakeCmoAssets();
+
+type MockSearchContainerProps = Pick<SearchContainerProps, "candidateCount">;
+
+const renderSearchContainer = ({
+  candidateCount,
+}: MockSearchContainerProps) => {
+  const mockUpdate = jest.fn();
+  const mockValues = jest.fn();
+  const mockSubmit = jest.fn();
+  return (
+    <>
+      {render(
+        <SearchContainer
+          classifications={mockClassifications}
+          cmoAssets={mockCmoAssets}
+          candidateCount={candidateCount}
+          updateCandidateFilter={mockUpdate}
+          updateInitialValues={mockValues}
+          handleSubmit={mockSubmit}
+        />,
+      )}
+    </>
+  );
+};
+
+describe("SearchContainer", () => {
+  it("should render request button with candidates", async () => {
+    renderSearchContainer({ candidateCount: 10 });
+    await expect(
+      screen.getByRole("button", { name: /request candidates/i }),
```

Comment:
Are we only writing tests for the English version of the site?

Reply:
Correct, the text in the button doesn't matter so much. If it were french we would just modify the query to look for "Demande de candidats" and it should still pass.

The text is just a way to query the screen for the button. A French test would look like:

```tsx
....
describe("SearchContainer", () => {
  it("should render request button with candidates", async () => {
    renderSearchContainer({ candidateCount: 10 });
    await expect(
      screen.getByRole("button", { name: /demande de candidats/i }),
    ).toBeInTheDocument();
  });
...
```